lib/openhab/core/rules/rule.rb

```diff
@@ -122,6 +146,7 @@ def status_info
         def inspect
           r = "#<OpenHAB::Core::Rules::Rule #{uid}"
           r += " #{name.inspect}" if name
+          r += " #{description.inspect}" if description
```

Comment:
I actually left description out of the inspect output on purpose. it's too noisy

Reply:
I've removed it.